app/assets/stylesheets/application.scss

```diff
@@ -63,6 +52,20 @@ main img {
   max-width: 100%;
 }
 
+footer .govuk-width-container {
```

Comment:
Now you have added a global width the following looks like it should be redundant:

```scss
footer .govuk-width-container {
  max-width: 1200px;
}
```